waiter/src/waiter/main.clj

```diff
@@ -117,6 +118,7 @@
     (reify Thread$UncaughtExceptionHandler
       (uncaughtException [_ thread throwable]
         (log/error throwable (str (.getName thread) " threw exception: " (.getMessage throwable))))))
+  (jvm-metrics/instrument-jvm)
```

Comment:
Inter-router metrics sync-ing is triggered via `waiter.metrics/get-core-metrics` which uses a `MetricFilter` that looks into service metrics only.